lib/collection.js

```diff
@@ -222,13 +218,7 @@ Collection.prototype.isCapped = function(cb) {
   });
 };
 
-Collection.prototype.stats = function(cb) {
```

Comment:
Why remove this method? This is a public method documented here. http://docs.mongodb.org/manual/reference/method/db.collection.stats/#db.collection.stats